WcaOnRails/app/assets/stylesheets/persons.scss

```diff
@@ -178,6 +178,12 @@
           color: $orange;
         }
       }
+
+      &.highlight-gold tr.gold-place,
+      &.highlight-silver tr.silver-place,
+      &.highlight-bronze tr.bronze-place {
+        background-color: #fefbd8;
```

Comment:
It would be nice to have this color code in the `variables.scss` file in the same directory, instead of hardcoded here.
That way if we ever need to use it again or change it, we don't have to look everywhere for where it's used!